### PR TITLE
Fixing Issue#308

### PR DESCRIPTION
**Pull request by rishubhjain**
tendrl-bug-id: Tendrl/ceph-integration#308

### CODE REVIEW

File: tendrl/ceph_integration/ceph.py [hunk 1]
```diff
@@ -51,6 +51,13 @@ class AdminSocketError(MonitoringError):
     pass
 
 
+class AdminSocketNotFoundError(MonitoringError):
+    """Scokets not found in /var/run/ceph.
```
Comment (shtripat) [inline tendrl/ceph_integration/ceph.py]: spelling `Scokets`

Comment (rishubhjain) [reply tendrl/ceph_integration/ceph.py]: Done :)

File: tendrl/ceph_integration/ceph.py [hunk 2]
```diff
@@ -668,6 +675,12 @@ def get_heartbeats():
     # FSID string to cluster name string
     fsid_names = {}
 
+    sock_files = glob("/var/run/ceph/*.asok")
+
+    if not sock_files:
+        # If Admin Sockets are not found 
+        raise AdminSocketNotFoundError
```
Comment (r0h4n) [inline tendrl/ceph_integration/ceph.py]: add a meaningful error msg 

raise AdminSocketNotFoundError(msg)

**Review by r0h4n — CHANGES_REQUESTED** [review]

File: tendrl/ceph_integration/ceph.py (hunk 2 above)
Comment (rishubhjain) [reply tendrl/ceph_integration/ceph.py]: Done

**Review by r0h4n — APPROVED** [review]

**Pull request merged**